app/src/main/java/org/mozilla/reference/browser/browser/BaseBrowserFragment.kt

```diff
@@ -161,6 +163,16 @@ abstract class BaseBrowserFragment : Fragment(), UserInteractionHandler, Activit
             owner = this,
             view = view
         )
+        shareDownloadsFeature.set(
+            ShareDownloadFeature(
+                context = requireContext().applicationContext,
+                httpClient = requireComponents.core.client,
+                store = requireComponents.core.store,
+                tabId = null
```

Comment:
```suggestion
                tabId = sessionId
```

Shouldn't this have the `sessionId`?

Reply:
Tbh, I don't really know. One further down, DownloadsFeature has an optional `tabId` parameter that isn't passed in, so it ends up being `null`, which is what I based this PR on.

But now I see that for PrompFeature we pass `sessionId` to the `customTabId` parameter and for `FullScreenFeature` we pass `sessionId` to `tabId`.

So... I don't know what would be correct here. I did test this again now wiith sharing from multiple tabs and both variants work. But then again maybe this is only relevant for whenyou switch tabs before the share completes or something which isn't somewthing I can easily manually test.

Reply:
I looked again into the fenix history around this and it seesm that the parameter inside `DownloadFeature` was renamed first from `sessionId` to `customTabId` and then further to `tabId`. This makes some more sense now. :+1: